testing/integration/upgrade_fleet_test.go

```diff
@@ -247,6 +393,7 @@ func testUpgradeFleetManagedElasticAgent(
 		nonInteractiveFlag = true
 	}
 	installOpts := atesting.InstallOpts{
+		Insecure:       true,
```

Comment:
Could this be passed as parameter so that we use insecure: true only on the new test ?

Reply:
I guess this line wasn't needed after all ? I cannot find it in the latest version of the PR